src/app/_components/writingPad/writingPad.tsx

```diff
@@ -1,47 +1,180 @@
 "use client";
-
-import ReactQuill from "react-quill";
+import EditNoteIcon from "@mui/icons-material/EditNote";
```

Comment:
I think we are naming components with Pascal casing, let call it WritingPad.tsx

Reply:
done